ui/components/Header/index.tsx

```diff
@@ -206,6 +206,40 @@ const Header = ({ currentUser, fetchingUser, group, round, bucket, ss }) => {
 
   const showJoinRoundButton = round && notAMember && allowedToJoinOrRequest;
 
+  useEffect(() => {
+    const showJoinRoundButton2 = round &&  // A copy to be evaluated on component mount
+      round.registrationPolicy !== "INVITE_ONLY" &&
```

Comment:
If this is for auto join. Then I dont think we need the invite only - as our type of event will be open for all

Reply:
Maybe im wrong here

Reply:
didn't saw the !==